src/renderer/geometry/instanced_mesh.rs

```diff
@@ -334,11 +555,20 @@ impl Geometry for InstancedMesh {
         color_texture: Option<ColorTexture>,
         depth_texture: Option<DepthTexture>,
     ) {
+        // Update the instance buffers if required.
+        let is_material_transparent = false; // is this correct? PostMaterial doesn't provide this.
```

Comment:
Need an expert opinion here, I don't know enough about what a `PostMaterial` is, or how it is used to determine whether we should default to transparent or opaque.